### PR TITLE
Fix of #229

### PR DESCRIPTION
**Pull request by cyberone**
Fix of https://github.com/yegor256/takes/issues/229 (PsGithub sends invalid POST request)

1. Removed request parameters from URI
1. Added request parameters to request body

### CODE REVIEW

**Comment by davvd** [comment]
@cyberone Thanks, I will find someone to review this PR soon

**Comment by davvd** [comment]
@carlosmiranda review this one, please

**Comment by carlosmiranda** [comment]
@cyberone Shouldn't there be a unit test that replicates the bug for this?

**Comment by cyberone** [comment]
@carlosmiranda there's already an issue https://github.com/yegor256/takes/issues/42 about missing test on PsGithub.

**Comment by carlosmiranda** [comment]
@cyberone In that case you should add a puzzle to test this particular scenario. We can't leave it as it is, or nobody will notice and your change will be untested.

**Comment by cyberone** [comment]
@carlosmiranda added puzzle. Please re-check.

File: src/main/java/org/takes/facets/auth/social/PsGithub.java [hunk 1]
```diff
@@ -51,6 +51,8 @@
  * @version $Id$
  * @since 0.1
  * @checkstyle MultipleStringLiteralsCheck (500 lines)
+ * @todo #229: PsGithubTest should check that POST request arguments are
```
Comment (carlosmiranda) [inline src/main/java/org/takes/facets/auth/social/PsGithub.java]: @cyberone Where is the estimated time? Should be (for example) `#229:30min`.

File: src/main/java/org/takes/facets/auth/social/PsGithub.java [hunk 2]
```diff
@@ -51,6 +51,8 @@
...
+ *  passed as parameters at request body, not at head.
```
Comment (carlosmiranda) [inline src/main/java/org/takes/facets/auth/social/PsGithub.java]: @cyberone I think you mean the request URL, not head.

**Comment by carlosmiranda** [comment]
@cyberone A couple of comments related to the puzzle above.

**Comment by cyberone** [comment]
@carlosmiranda done

**Comment by carlosmiranda** [comment]
@cyberone Thanks!

**Comment by carlosmiranda** [comment]
@rultor merge please

**Comment by yegor256** [comment]
@rultor merge

**Comment by davvd** [comment]
@elenavolokhova please, review this task for compliance with our [quality rules](http://www.teamed.io/qa.html)

**Comment by davvd** [comment]
@rultor deploy pls

**Comment by elenavolokhova** [comment]
@carlosmiranda
 According to our [quality rules](http://www.teamed.io/qa.html):
 > Pull request title explains the problem it is fixing.
 
 The title of this request doesn't contain such information. As far as you are the performer of this ticket you are responsible for its quality. 
Please fix this issue and make sure that everyone follows this rule in your tickets in the future.

According to our [quality rules](http://www.teamed.io/qa.html):
 > Comments were mostly about design problems, not cosmetic issues.
 
 There were mostly cosmetic issues found during review. Try to pay attention to bigger design problems and the overall quality of the code.
Please confirm that this new rule is clear.

**Comment by carlosmiranda** [comment]
@cyberone please rename the ticket title to reflect the actual problem as requested by @elenavolokhova .

**Comment by cyberone** [comment]
@carlosmiranda done.

**Comment by elenavolokhova** [comment]
@cyberone Thank you

**Comment by elenavolokhova** [comment]
@davvd Quality is acceptable.

**Comment by davvd** [comment]
> @davvd Quality is acceptable.

@elenavolokhova thanks for the review, we'll try better next time

**Comment by davvd** [comment]
> @davvd Quality is acceptable.

@elenavolokhova thanks for the review, we'll try better next time

**Comment by davvd** [comment]
> @davvd Quality is acceptable.

@elenavolokhova thanks a lot, next time everybody should try to make it better

**Comment by davvd** [comment]
> @davvd Quality is acceptable.

@elenavolokhova thanks, got it. everybody, please try to make it good next time

**Comment by davvd** [comment]
> @davvd Quality is acceptable.

@elenavolokhova thanks for the review, we'll try better next time

**Comment by davvd** [comment]
> @davvd Quality is acceptable.

@elenavolokhova thank you, let's try to make it "good" next time

**Comment by davvd** [comment]
@carlosmiranda I added 10 mins to @elenavolokhova (for QA review) in transaction 56879541

I just added **21 mins** to your account, many thanks for your contribution.. 56879557 spent here

you're getting extra minutes here (c=6)

+21 added to your rating, current score is: [+4501](http://www.netbout.com/b/35023?open=rating)

**Pull request merged**